base/logging.h

```diff
@@ -82,16 +82,27 @@ void OutputDebugStringUTF8(const char *p);
 
 #ifdef _WIN32
 
+#define _LOG(s, ...) { \
```

Comment:
As long as it builds everywhere I'm happy, but `_LOG` makes me worry it'll overlap with something on some platform.  Maybe `NATIVE_LOG` or `LOG_IMPL` or something would be safer.

-[Unknown]